docs/CONFIG.md

```diff
@@ -9243,6 +9243,33 @@ TipCapMinimum is the minimum gas tip to use when submitting transactions to the
 
 (Only applies to EIP-1559 transactions)
 
+## EVM.GasEstimator.DAOracle
+```toml
+[EVM.GasEstimator.DAOracle]
+OracleType = 'OP' # Example
+OracleAddress = '0x420000000000000000000000000000000000000F' # Example
+CustomGasPriceAPICalldata = '' # Default
+```
+
+
+### OracleType
+```toml
+OracleType = 'OP' # Example
```

Comment:
as noted, we should add these to chain default tomls, examples: https://github.com/smartcontractkit/chainlink/blob/develop/ccip/config/evm/Mantle_Sepolia.toml#L2

you can search by chain type and for each chain type, add the corresponding DA oracle type

confirm with @smartcontractkit/bix-ship